data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
 
 
 
+mission "Remnant: Tech Retrieval"
+	name "Remnant Human Tech Retrieval"
+	description "The Remnant are interested in learning what advances humanity has made over the past centuries. Bring them samples of advanced technology."
+	source "Viminal"
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 30
+	on offer
+		"remnant taely" ++
```

Comment:
Should be `"remnant met taely"`. Or, unless you plan on having Taely be introduced anywhere else, this condition can just be removed and the log for Taely can go here instead.

Reply:
Fixed. Thanks! (I'm also using it as a counter for how many times the player has done a mission that Taely would be concerned with)